pages/index.vue

```diff
@@ -29,17 +31,21 @@ export default Vue.extend({
   },
   head() {
     return {
-      title: "ReplicaZone - Snipcart Demo",
+      title: this.$i18n.t('full_title').toString()
     };
   },
-  async asyncData({ route }) {
+  async asyncData() {
     return {
-      products: Products,
+      products: {}
```

Comment:
You can access i18n in the context provided by nuxt to asyncData
```
asyncData({ i18n }) {
  return {
    products: i18n.locale === 'fr' ? ProductsFR : Products
  }
}
```

Reply:
Didn't know, thanks!